### PR TITLE
Added deno 🦕

### PR DESCRIPTION
**Pull request by lunaroyster**
Why
===

We decided to add deno: [internal slack link]

What changed
============

Added deno as a language on polygott.

Test plan
=========

1. Installed deno on polygott: (this)
2. Installed deno on eval server
3. Installed deno on frontend
4. Ran deno code. It runs!

### CODE REVIEW

File: languages/deno.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name = "deno"
+entrypoint = "index.ts"
+extensions = [
+  "ts"
+]
+packages = [
+]
+setup = [
+  "curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/usr sh"
```
Comment (kochman) [inline languages/deno.toml:9]: I take it there's no Ubuntu deb or similar? I'm not a fan of install scripts but at quick glance this seems to be the only official way to install Deno...

Comment (kochman) [reply languages/deno.toml:9]: It looks like their install script also takes an arg to grab a specific version—we probably want to do this so rebuilding polygott doesn't implictly upgrade everyone.

Comment (lunaroyster) [reply languages/deno.toml:9]: I couldn't find one either. I'll ask around on their discord.

Comment (lunaroyster) [reply languages/deno.toml:9]: Ah thanks, I'll fix the version in place

**Comment by lunaroyster** [comment]
:sauropod:!

**Pull request merged**